src/akamai_cache_buster/bustCache.py

```diff
@@ -55,22 +55,44 @@ def main():
     
     #connect to akamai and validate
     initEdgeGridAuth()
+        
 
-    print(getYMLFromUrl("https://cloud.redhat.com/config/main.yml").get(appName).get("frontend").get("paths"))
+    paths = getYMLFromUrl("https://cloud.redhat.com/config/main.yml").get(appName).get("frontend").get("paths")
+    for path in paths:
+        print(path.split('/'))
+   
+    #create a request for each path
+    for path in paths:
 
-    urls = []
-    for paths in getYMLFromUrl("https://cloud.redhat.com/config/main.yml").get(appName).get("frontend").get("paths"):
-        urls.append("https://cloud.redhat.com" + paths)
-    
-    for url in urls:
-        print(url)
+        #create the basic metadata message
+        metadata = '<?xml version=\"1.0\"?>\n'
```

Comment:
A real Pythonista would know way better than me... but there is some better way to build this string I think.
1) You could use a real XML generator
2) I think there is an equivalent of the JavaScript
`
<?xml version=\"1.0\"?>
${someVariableHere}
`

Pattern.

Reply:
Like dis for instance:
```python
[iphands@handslap ~] python
Python 3.7.4 (default, Jul 31 2019, 14:55:20) 
[GCC 9.1.1 20190730 gcc-9-branch@273913] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> """
... This is a test
... {test}
... Blahblah
... {test2}
... """.format(test="foo", test2="bar")
'\nThis is a test\nfoo\nBlahblah\nbar\n'
```

Reply:
I was about to suggest `format()` - I think that's a nice clean way to go about this.

Reply:
Not sure how relevant it'll be here, or if it'll even be relevant... but, you can actually build long multi line comments in Python with triple quotes `"""`. I had to use that in another life...

Reply:
triple quotes will either cause indentation problems in the final XML or look really gross in the source so I opted to not use them